### DISCUSSION TITLE
Using Home assistant entities as data sources for custom pages

### DISCUSSION THREAD

**Discussion by djgalati** [gickowtf/pixoo-homeassistant#8] [Q&A] [answered]
I was curious if there is a way to build a a custom page using sensors from home assistant. I tried using: `text: "{{sensor.sensor_name}}"` but it shows as template error. Any ideas on how to resolve my issue? 

Thanks in advance for any help

**Comment by gickowtf** [comment, marked as answer]
Pls try "{{ states.sensor.sensor_name.state }}"

**Comment by djgalati** [comment]
That worked!

would it be safe to assume most sensors should work with `"{{states.(home assistant entity name).state}}"`? I have some binary sensors and other types I'd like to get added and I just want to make sure!

**Reply by djgalati** [reply]
Another thing I'm curious about, What if I have a secondary server I'd like to pull stats from and display at the same time?
 e.g. a second raspberry pi running a different set of servers, would it be possible to use data it(home assistant pi) is able to pull from a ssh command on the second pi in order to display that data as well or would that need to be an added feature?

**Reply by SB2DD** [reply]
I don't think this is in the scopes of this integration. Instead, I would maybe use another integration in Home Assistant that would be able to get the data through ssh or other means (perhaps, maybe think of MQTT) and expose it to an entity, and use this integration to show that data on the display.

**Reply by djgalati** [reply]
That's a fantastic idea. I will pursue this!

**Comment by djgalati** [comment]
With all the recent updates, I notice my config file keeps changing the content field to the following. Is there something I missed in getting this updated?

`content:
        "[object Object]": null`

my code was:

`content: {{states.sensor.processor_temperature.state}}`

**Reply by SB2DD** [reply]
Make sure to write
`content: "{{states.sensor.processor_temperature.state}}"` (The " are very important! (or else HA freaks out and shows the error you're seeing..))

**Reply by djgalati** [reply]
That was it. Man I feel like a total noob sometime.

Thank you!
